EASI.postman_collection.json

```diff
@@ -2066,7 +2066,7 @@
 								"body": {
 									"mode": "graphql",
 									"graphql": {
-										"query": "mutation UpdateRecommendationOrder($input: UpdateTRBGuidanceLetterRecommendationOrderInput!) {\r\n    updateTRBGuidanceLetterRecommendationOrder(input: $input) {\r\n        id\r\n        title\r\n        recommendation\r\n    }\r\n}",
+										"query": "mutation UpdateRecommendationOrder($input: UpdateTRBGuidanceLetterRecommendationOrderInput!) {\r\n    updateTRBGuidanceLetterRecommendationOrder(input: $input) {\r\n        id\r\n        title\r\n        recommendation\r\n        category\r\n    }\r\n}",
 										"variables": "{\r\n    \"input\": {\r\n        \"trbRequestId\": \"{{trbRequestID}}\",\r\n        \"newOrder\": [\r\n            \"0a277d1c-0c64-4a5b-9d22-2d8c01e98fa6\",\r\n            \"898065c3-0ade-4309-bb9d-21ee9911acff\",\r\n            \"52de3093-86c0-4076-9410-6333345a8bbf\"\r\n        ]\r\n    }\r\n}"
```

Comment:
These variables throw an error because it's missing the "category" field. Also the other queries in Postman need to have their TRB "recommendations" fields changed to "insights".

Also, when re-ordering, because Category is passed outside of the array, does that make it so that all items become the single passed-in category?

Reply:
updated! thanks

and no, the `category` property passed in when reordering is just telling the BE which category is being reordered